tests/integration/test_client.py

```diff
@@ -4,7 +4,11 @@
 
 from amazon_transcribe.model import TranscriptEvent
 from amazon_transcribe.client import TranscribeStreamingClient
-from amazon_transcribe.exceptions import BadRequestException
+from amazon_transcribe.exceptions import (
+    BadRequestException,
+    UnknownServiceException,
```

Comment:
minor nit, but it's generally good practice to alphabetize imports for when the lists get longer.